### DISCUSSION TITLE
interaction.channel.send() does not work, interaction.followup.send() does work!

### DISCUSSION THREAD

**Discussion by MCilento93** [nextcord/nextcord#1155] [Q&A]
My bot: https://github.com/MCilento93/Crown-Store-deals-bot

In my servers the bot works perfectly, in all its features.

The same invite link was sent to a more structured server, with role-permissions. 
In this structured server the bot is allowed to "send message" infact, the functions with **`interaction.followup.send()` work perfectly**, whereas defs with **`interaction.channel.send()` do not work** if neither me nor the admin invokes the slash command.

Do the two functions requires different permissions?

```
Ignoring exception in command <nextcord.application_command.SlashApplicationCommand object at 0x7f710d8a3c70>:
Traceback (most recent call last):
  File "/home/runner/Crown-Store-deals-bot/.pythonlibs/lib/python3.10/site-packages/nextcord/application_command.py", line 918, in invoke_callback_with_hooks
    await self(interaction, *args, **kwargs)
  File "/home/runner/Crown-Store-deals-bot/main.py", line 88, in crown_store_discounts
    await interaction.channel.send(chunk)
  File "/home/runner/Crown-Store-deals-bot/.pythonlibs/lib/python3.10/site-packages/nextcord/abc.py", line 1565, in send
    data = await state.http.send_message(
  File "/home/runner/Crown-Store-deals-bot/.pythonlibs/lib/python3.10/site-packages/nextcord/http.py", line 393, in request
    raise Forbidden(response, data)
nextcord.errors.Forbidden: 403 Forbidden (error code: 50001): Missing Access

The above exception was the direct cause of the following exception:

nextcord.errors.ApplicationInvokeError: Command raised an exception: Forbidden: 403 Forbidden (error code: 50001): Missing Access
```

**Comment by MCilento93** [comment]
Checking permissions in the channel in which `interaction.channel.send()` troubles, I have realized that "send_message" is set to False.
`interaction.followup.send()` is a weekbook, whereas  `interaction.channel.send()` not. Do they require different permissions to work?

`
{'create_instant_invite': False, 'kick_members': False, 'ban_members': False, 'administrator': False, 'manage_channels': False, 'manage_guild': False, 'add_reactions': True, 'view_audit_log': False, 'priority_speaker': False, 'stream': False, 'read_messages': True, 'send_messages': False, 'send_tts_messages': False, 'manage_messages': False, 'embed_links': False, 'attach_files': False, 'read_message_history': True, 'mention_everyone': False, 'external_emojis': False, 'view_guild_insights': False, 'connect': False, 'speak': False, 'mute_members': False, 'deafen_members': False, 'move_members': False, 'use_voice_activation': False, 'change_nickname': True, 'manage_nicknames': True, 'manage_roles': False, 'manage_webhooks': False, 'manage_emojis': True, 'use_slash_commands': False, 'request_to_speak': False, 'manage_events': False, 'manage_threads': False, 'create_public_threads': True, 'create_private_threads': True, 'external_stickers': True, 'send_messages_in_threads': False, 'start_embedded_activities': True, 'moderate_members': False} 
`

**Reply by ooliver1** [reply]
Sending to a channel requires permissions yes. `followup` is a webhook so it is allowed to send. If a mod/admin wanted to disable this they would disable `use_slash_commands`.
